src/main/java/me/totalfreedom/totalfreedommod/command/Command_nicknyan.java

```diff
@@ -9,7 +9,7 @@
 import org.bukkit.entity.Player;
 
 @CommandPermissions(level = Rank.OP, source = SourceType.ONLY_IN_GAME)
-@CommandParameters(description = "Essentials Interface Command - Randomize the colors of your nickname.", usage = "/<command> <<nick> | off>")
+@CommandParameters(description = "Essentials Interface Command - Randomize the colors of your nickname.", usage = "/<command> <[nick] | off>")
```

Comment:
Can you drop the first chunk of this description as it just doesn't read right...

Reply:
Yeah, Pretty easy change